app/brave_main_delegate.cc

```diff
@@ -158,7 +172,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
 
   // Brave's sync protocol does not use the sync service url
   command_line.AppendSwitchASCII(switches::kSyncServiceURL,
-                                 "https://no-thanks.invalid");
+                                 kBraveSyncServiceURL);
```

Comment:
we should check to make sure the sync url is empty before setting it, otherwise you can't override

Reply:
```
void BraveCommandLineHelper::AppendSwitch(const char* switch_key) {
  if (!command_line_.HasSwitch(switch_key))
    command_line_.AppendSwitch(switch_key);
}
```
our BraveCommandLineHelper already checks it